meson.build

```diff
@@ -12,13 +12,13 @@ taglib = dependency('taglib')
 subdir('src')
 subdir('po')
 
-executable('org.nickvision.tagger', sources, dependencies: [threads, adwaita, jsoncpp, curlpp, taglib], install: true, install_mode: 'rwxrwxrwx')
-install_data('fpcalc', install_dir: 'bin', install_mode: 'rwxrwxrwx')
+executable('org.nickvision.tagger', sources, dependencies: [threads, adwaita, jsoncpp, curlpp, taglib], install: true, install_mode: 'rwxr-xr-x')
+install_data('fpcalc', install_dir: 'bin', install_mode: 'rwxr-xr-x')
```

Comment:
Is the default executable mode not good enough?

You only need an install_mode for `install_data('fpcalc', ...)` because it differs from the mode of the file in git / source archives.

Reply:
That is correct, if this parameter is not needed it should be dropped.